### PR TITLE
fix fluentbit naming and release

### PR DESCRIPTION
**Pull request by hithwen**
Also fix install instructions

### CODE REVIEW

File: fluent_bit/README.md [hunk 1]
```diff
@@ -18,18 +18,17 @@ Follow the instructions below to install and configure this check for an Agent r
...
 
-1. Install the [developer toolkit][12] on any machine.
+For Agent v7.21+ / v6.21+, follow the instructions below to install the Sendmail check on your host. See [Use Community Integrations][3] to install with the Docker Agent or earlier versions of the Agent.
 
-2. Run `ddev release build fluent_bit` to build the package.
+1. Run the following command to install the Agent integration:
 
-3. [Download the Datadog Agent][3].
+   ```shell
+   datadog-agent integration install -t datadog-fluentbit==<INTEGRATION_VERSION>
```
Comment (yzhan289) [inline fluent_bit/README.md:28]: Not sure if this should be `fluentbit` or `fluent-bit`
```suggestion
   datadog-agent integration install -t datadog-fluent-bit==<INTEGRATION_VERSION>
```

File: fluentbit/README.md [hunk 2]
```diff
@@ -18,18 +18,17 @@ Follow the instructions below to install and configure this check for an Agent r
 
 ### Installation
 
-To install the Fluent Bit check on your host:
-
+### Installation
 
-1. Install the [developer toolkit][12] on any machine.
+For Agent v7.21+ / v6.21+, follow the instructions below to install the Sendmail check on your host. See [Use Community Integrations][3] to install with the Docker Agent or earlier versions of the Agent.
```
Comment (hithwen) [inline fluentbit/README.md:23]:
```suggestion
For Agent v7.21+ / v6.21+, follow the instructions below to install the Fluent Bit check on your host. See [Use Community Integrations][3] to install with the Docker Agent or earlier versions of the Agent.
```

**Comment by yzhan289** [comment]
You will also need to change fluent_bit in the Azure Pipelines: https://github.com/DataDog/integrations-extras/blob/82afbd9e429f8d1e30ba492c8112ba4021f2cc57/.azure-pipelines/all.yml#L62

**Comment by yzhan289** [comment]
codecov.yaml also needs to be updated: https://github.com/DataDog/integrations-extras/blob/e870a239e6a2d9f1afc3d092e707a55e094674a4/.codecov.yml#L42

**Review by kayayarai — APPROVED** [review]

**Comment by hithwen** [comment]
@zhenyami, FYI, I had to do some renaming since it was causing some issues in our end. The tile should be up tomorrow

**Pull request merged**